admin_tests/pre_reg/test_views.py

```diff
@@ -25,7 +25,8 @@
 
 
 class TestDraftListView(AdminTestCase):
-    def setUp(self):
+    @mock.patch('admin.pre_reg.views.DraftRegistration.get_metadata_files')
```

Comment:
I don't need to have all the file data in the draft registration for all of these operations, yet I must submit for review which now does a `get_metadata_files` in setup. Therefore it is mocked in setup.